howso/analysis.amlg

```diff
@@ -62,6 +63,8 @@
 			;{type "boolean"}
 			bypass_calculate_feature_weights (null)
 			;{type "boolean"}
+			bypass_calculate_prediction_stats (null)
```

Comment:
maybe this should default to false? rather than be optional

Reply:
Res also says we should rename to "bypass_CACHE_prediction_stats" (capitalized for emphasis on the diff)